test/end-to-end/reports/report_accounts/report_accounts.page.js

```diff
@@ -0,0 +1,58 @@
+/* global browser, element, by */
+
+const chai = require('chai');
+const helpers = require('../../shared/helpers');
+
+helpers.configure(chai);
+
+const FU = require('../../shared/FormUtils');
+const ReportPage = require('../page');
+const components = require('../../shared/components');
+
+class AccountReportPage {
```

Comment:
Cool :+1:

Considering that this page will look identical for every report, I propose that you refactor this so that the common elements can be shared in a generic `ReportPage`.  For example - saving and printing will be the same for all reports.